pipeline/pipelines-ui/base/deployment.yaml

```diff
@@ -13,10 +13,21 @@ spec:
       labels:
         app: ml-pipeline-ui
     spec:
+      volumes:
```

Comment:
This would also need to be a GCP overlay which means you would also need to modify 
https://github.com/kubeflow/manifests/blob/127dacce625e5dfc7630302ea82cf8ad56f9b6dd/kfdef/kfctl_gcp_iap.yaml#L278

and
https://github.com/kubeflow/manifests/blob/127dacce625e5dfc7630302ea82cf8ad56f9b6dd/kfdef/kfctl_gcp_basic_auth.yaml#L270

Reply:
Thanks! Made requested changes.